plugins/pdr_deterministic_plugin/ufm_sim_web_service/isolation_mgr.py

```diff
@@ -112,12 +112,16 @@ def __init__(self, ufm_client: UFMCommunicator, logger):
         self.temp_check = pdr_config.getboolean(Constants.CONF_COMMON,Constants.CONFIGURED_TEMP_CHECK)
         self.no_down_count = pdr_config.getboolean(Constants.CONF_COMMON,Constants.NO_DOWN_COUNT)
         self.access_isolation = pdr_config.getboolean(Constants.CONF_COMMON,Constants.ACCESS_ISOLATION)
+        self.test_mode = pdr_config.getboolean(Constants.CONF_COMMON,Constants.TEST_MODE)
         # Take from Conf
         self.logger = logger
-        intervals = [x[0] for x in Constants.BER_THRESHOLDS_INTERVALS]
+        self.ber_intervals = Constants.BER_THRESHOLDS_INTERVALS
+        if self.test_mode:
+            self.ber_intervals = [[0.5 * 60, 3]]
```

Comment:
?
a. if not test mode - you take constant
   if test mode - you take another constant.
b. self.ber_intervals = Constants.BER_THRESHOLDS_INTERVALS not self.test_mode else [[0.5 * 60, 3]]

Reply:
That correct,
a. test mode is more rapid and thus I changed the values to reduced.
b. Changed it to that